ensure_tiller_installed.go

```diff
@@ -22,16 +22,16 @@ import (
 // for it to start. A service account and cluster role binding are also created.
 // As a first step, it checks if Tiller is already ready, in which case it
 // returns early.
-func (c *Client) EnsureTillerInstalled(ctx context.Context) error {
-	return c.EnsureTillerInstalledWithValues(ctx, []string{})
+func (c *Client) EnsureTillerInstalled(ctx context.Context, upgrading bool) error {
```

Comment:
Won't this change the signature? 

So we'd need to change it everywhere we call `EnsureTillerInstalled`? This is used in lots of e2e tests. What about adding it as an optional parameter? 

https://github.com/giantswarm/helmclient/blob/d3c47349776d30db46c3e200ee94d16aeb22aa0b/helmclient.go#L55-L59

We could default it to false and only enable for draughtsman and chart-operator where we want the upgrade to happen.

Reply:
that's nice, I'm moving it from a parameter to config. PTAL!